src/main/kotlin/no/nav/syfo/cronjob/preloadcache/PreloadCacheCronjob.kt

```diff
@@ -0,0 +1,80 @@
+package no.nav.syfo.cronjob.preloadcache
+
+import kotlinx.coroutines.runBlocking
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.client.veiledertilgang.VeilederTilgangskontrollClient
+import no.nav.syfo.cronjob.Cronjob
+import no.nav.syfo.cronjob.CronjobResult
+import no.nav.syfo.personstatus.db.hentUbehandledePersonerTilknyttetEnhet
+import org.slf4j.LoggerFactory
+import java.time.*
+
+class PreloadCacheCronjob(
+    private val database: DatabaseInterface,
+    private val tilgangskontrollClient: VeilederTilgangskontrollClient,
+) : Cronjob {
+    private val log = LoggerFactory.getLogger(PreloadCacheCronjob::class.java)
+    private val runAtHour = 10
+    private val chunkSize = 50
+
+    override val initialDelayMinutes: Long = calculateInitialDelay()
+    override val intervalDelayMinutes: Long = 60L * 24
+
+    override suspend fun run() {
+        runJob()
+    }
+
+    fun runJob(): CronjobResult {
+        log.info("Run PreloadCacheCronjob")
+        val result = CronjobResult()
+
+        database.getEnheter()
+            .forEach { enhetNr ->
+                try {
+                    val personer = database.hentUbehandledePersonerTilknyttetEnhet(enhetNr)
```

Comment:
Her får vi vel også med oss noen personer uten aktiv oppgave (vi gjør noe ekstra filtrering der denne kalles fra `PersonoversiktStatusService.hentPersonoversiktStatusTilknyttetEnhet`), men det gjør kanskje ikke noe om vi cacher litt for mange?